Rover/mode_circle.cpp

```diff
@@ -70,7 +70,7 @@ bool ModeCircle::set_center(const Location& center_loc, float radius_m, bool dir
     return true;
 }
 
-// initialize dock mode
+// initialize circle mode from current position
 bool ModeCircle::_enter()
```

Comment:
I would have expected a `tracking_back = false;` in here somewhere.

Reply:
Fixed